.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Related Ticket: TINY-
+
+Description of Changes:
+* Placeholder text
+
+DOD:
```

Comment:
DOD?

Reply:
Copied that from the DX and Docs template tbh :shrug: I'm not sure what it actually means.

Reply:
Ahh, it's `definition of done`

Reply:
As discussed on slack, changed this to "Pre-checks" and added a couple other items